dlh_utils/flags.py

```diff
@@ -270,8 +270,7 @@ def flag_summary(df, flags=None, pandas=False):
 
         flags_out.append((df
                           .select(col)
-                         .where(F.col(col) is True)
```

Comment:
Tested both ways, neither actually does what is intended
Intention of this line of code appears to be to count number of non-null rows
Fixed in next push to unit_test_refresh

Reply:
Just had another review of docstrings and it's actually just to count booleans.
In this case, this is fine! 👍

Reply:
Had another review of docstrings and actually, this is fine! :)